### DISCUSSION TITLE
Material dock not showing

### DISCUSSION THREAD

**Discussion by Guurf** [blackears/cyclopsLevelBuilder#91] [Announcements]
When 'CyclopsBlock' is selected it shows the Face Properties and the Tool Properties but i can't find the material dock anywhere.

**Comment by blackears** [comment]
If you're using the latest development version, this dock has been moved to the lower middle panel.

**Comment by nikixos** [comment]
I had this issue as well, Materials is in the lower panel beside Output lol. What does Cyclops > Enable Cyclops do?

**Comment by blackears** [comment]
It makes the toolbar buttons always on so that you don't have to select a block first before using them, but it doesn't work very well.  It's meant to be a way to get around Godot's lack of a true tools API.
